src/app_service/service/transaction/service.nim

```diff
@@ -169,6 +170,7 @@ QtObject:
         if tokenMetadata.isOwnerToken:
           let status = if receivedData.success: ContractTransactionStatus.Completed else: ContractTransactionStatus.Failed
           self.events.emit(SIGNAL_OWNER_TOKEN_SENT, OwnerTokenSentArgs(chainId: receivedData.chainId, txHash: receivedData.transactionHash, tokenName: tokenMetadata.tokenName, status: status))
+        self.events.emit(SIGNAL_TRANSACTION_SENDING_COMPLETE, receivedData)
```

Comment:
@stefandunca I feel there was a newer status-go event for pendingTx status updates. Is `PendingTransactionTypeDto.WalletTransfer.event` the right one to use?

Reply:
I see we have `const EventPendingTransactionUpdate*: string = "pending-transaction-update"` but it's not used anywhere. Not sure if the other one is planned for deprecation or something

Reply:
hmm not sure, but the event Ive connected to works correctly.

Reply:
> I see we have const EventPendingTransactionUpdate*: string = "pending-transaction-update" but it's not used anywhere. Not sure if the other one is planned for deprecation or something

This is a leftover from before introducing sessions in activity history. They are now handled in status-go in the activity session implementation so we can remove them.

Thanks for spotting it @dlipicar